frontend/src/components/fields/amount/Amount.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {AmountProps} from "./props";
+import {TextField} from "@mui/material";
+import {ValidationErrorMessages} from "../../../common/constansts/validationErrorMessages";
+import {LOWER_AMOUNT_LIMIT, UPPER_AMOUNT_LIMIT} from "../../../common/constansts/validation";
+
+const Amount: React.FC<AmountProps> = ({updateAmountField, updateIsValidAmountField}) => {
+
+    const handleAmountChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const inputValue = parseFloat(e.target.value);
+        const isValid = validateAmount(inputValue);
+        updateIsValidAmountField(isValid);
+        if (!isValid) return;
+        updateAmountField(inputValue);
+    };
+
+    const validateAmount = (amount: number): boolean => amount > LOWER_AMOUNT_LIMIT && amount <= UPPER_AMOUNT_LIMIT;
+
+    return (
+        <TextField
+            helperText={ValidationErrorMessages.AMOUNT_NOT_VALID}
```

Comment:
I think we can do same for all other inputs. When everything is read, for me, it's not user friendly. But if we show helper message all the time, it looks exactly as a helper message, not error. If you do so, you can change enum to InputHelperMessage